ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-bushslicer.yaml

```diff
@@ -0,0 +1,245 @@
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: LOCAL_IMAGE_BIN
+  required: true
+- name: IMAGE_SYNC
+  required: true
+- name: IMAGE_LOGBRIDGE
+  required: true
+- name: NAMESPACE
+  required: true
+- name: CLUSTER_TYPE
+  value: "azure"
+  required: true
+- name: TEST_IN_PRODUCTION
+- name: DEPLOY_VERSION
+  value: "v3.11"
+
+objects:
+
+# We want the cluster to be able to access these images
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-image-puller
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:unauthenticated
+
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: ${JOB_NAME_SAFE}
+    namespace: ${NAMESPACE}
+    annotations:
+      # we want to gather the teardown logs no matter what
+      ci-operator.openshift.io/wait-for-container-artifacts: teardown,test
+      ci-operator.openshift.io/containers-logged-on-failure: setup,test,teardown
+  spec:
+    restartPolicy: Never
+    activeDeadlineSeconds: 10800
+    terminationGracePeriodSeconds: 600
+    volumes:
+    - name: artifacts
+      emptyDir: {}
+    - name: shared-tmp
+      emptyDir: {}
+    - name: cluster-secrets-azure
+      secret:
+        secretName: azure
+    containers:
+    - name: test
+      image: quay.io/kwoodson/bushslicer
+      resources:
+        requests:
+          cpu: 1
+          memory: 300Mi
+        limits:
+          cpu: 3
+          memory: 4Gi
+      volumeMounts:
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      env:
+      - name: ARTIFACT_DIR
+        value: /tmp/artifacts
+      - name: HOME
+        value: /tmp/shared/home
+      - name: TEST_IN_PRODUCTION
+        value: ${TEST_IN_PRODUCTION}
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        # error handling and sync code
+        set -euo pipefail
+
+        trap 'touch /tmp/shared/exit' EXIT
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        mkdir -p "${HOME}"
+
+        # wait until the setup job creates admin.kubeconfig
+        while true; do
+          if [[ ! -f  /tmp/shared/_data/_out/admin.kubeconfig ]]; then
+            sleep 15 & wait
+            continue
+          fi
+          # if we got admin kubeconfig file with failure marker, ignore. Teardown is in progress. 
+          if [[ -f  /tmp/shared/exit ]]; then
+             exit 1
+          fi
+          break
+        done
+        echo "Found shared kubeconfig"
+
+        # test specific code
+        # don't let clients impact the global kubeconfig
+        #cp -r /tmp/shared/_data /go/src/github.com/openshift/openshift-azure/
+        #        cd /go/src/github.com/openshift/openshift-azure/
+        #        make e2e
+        #
+        /usr/bin/scl enable rh-git29 rh-ror50 /opt/rh/rh-ruby24/root/usr/local/bin/cucumber \
```

Comment:
This will be the command we decide on.  For now I'm building this and pushing it out to my quay.io location until we figure out where this will land.

Reply:
Instead of creating a net-new template, make this a script under a `make` target in openshift-azure, then update the `test` image in the original e2e template configurable and in your prowjob pass the image you want to use for `test` and potentially the `make` target via a different env variable (eg. `EXEC`).

Reply:
Instead of creating a net-new template, make this a script under a make target in openshift-azure, then update the test image in the original e2e template configurable and in your prowjob pass the image you want to use for test and potentially the make target via a different env variable (eg. EXEC).

Reply:
TEST_COMMAND